cmd/version.go

```diff
@@ -49,8 +50,19 @@ func newVersionCmd(homedir, getIstioVersion string) *cobra.Command {
 			}
 
 			logger.Infof("getistio version: %s\nactive istioctl: %s\n", getIstioVersion, cur.ToString())
+			k8sCLient, err := util.GetK8sClient()
+			if err != nil {
+				logger.Infof("no active Kubernetes clusters found\n")
+			} else {
+				v, err := k8sCLient.ServerVersion()
```

Comment:
err is shadowed here, so even if err is not nil, I think we end up entering the following `if err == nil & remote` block.